protocol-units/bridge/integration-tests/tests/eth_movement.rs

```diff
@@ -38,10 +38,12 @@ async fn test_movement_client_build_and_fund_accounts() -> Result<(), anyhow::Er
 	let faucet_client = movement_client.faucet_client().expect("Failed to get // FaucetClient");
 	let alice = LocalAccount::generate(&mut rand::rngs::OsRng);
 	let bob = LocalAccount::generate(&mut rand::rngs::OsRng);
+	let movement_client = movement_client.signer();
 
 	println!("\n=== Addresses ===");
 	println!("Alice: {}", alice.address().to_hex_literal());
 	println!("Bob: {}", bob.address().to_hex_literal());
+	println!("Movement Client: {}", movement_client.address().to_hex_literal());
```

Comment:
Any reason why we need to have the `println` here?  Seems to be debugging code of some sort so probably best to remove.

Reply:
Yep removing all debug code now.